shared/templates/template_OVAL_coreos_kernel_option

```diff
@@ -3,32 +3,32 @@
     <metadata>
       <title>Ensure that the most recent (default) CoreOS boot loader entry is configured to run Linux operating system with argument {{{ ARG_NAME_VALUE }}}</title>
       {{{- oval_affected(products) }}}
-      <description>Ensure {{{ ARG_NAME_VALUE }}} option is configured in the 'options' line in /boot/loader/entries/ostree-2-*.conf (or ostree-1-*.conf if the second version does not exists).</description>
+      <description>Ensure {{{ ARG_NAME_VALUE }}} option is configured in the 'options' line in /boot/loader/entries/ostree-2-.*.conf (or ostree-1-.*.conf if the second version does not exists).</description>
```

Comment:
Could you consider getting rid of this metadata boilerplate similarly to e.g. https://github.com/ComplianceAsCode/content/blob/master/shared/templates/template_OVAL_file_owner i.e. by using the `oval_metadata` macro? That would look like `{{{ oval_metadata("Ensure " + ARG_NAME_VALUE + " option is configured in the 'options' line in /boot/loader/entries/ostree-2-.*.conf (or ostree-1-.*.conf if the second version does not exist).") }}}`